dbeaver/dbeaver
# Add support for TiDB

Add support for TiDB

This adds icons, default ports and JDBC properties for TiDB.

To test:
- use a local [TiUP Playground](https://tiup.io)
- use the [TiDB Cloud](https://tidbcloud.com) developer tier.
- Build https://github.com/pingcap/tidb and run `./bin/tidb-server` and connect with `root` on `127.0.0.1:4000`

The only missing functionality I found is the explain output as the TiDB explain format isn't following the MySQL format. See [this document](https://docs.pingcap.com/tidb/dev/sql-statement-explain) for details on the format.
